content/blog/learning-to-manage-ilo-7-with-restful-interface-tool-ilorest-6-0.md

```diff
@@ -0,0 +1,757 @@
+	mso-tstyle-colband-size:0;
+	mso-style-noshow:yes;
+	mso-style-priority:99;
+	mso-style-parent:"";
+	mso-padding-alt:0in 5.4pt 0in 5.4pt;
+	mso-para-margin-top:0in;
+	mso-para-margin-right:0in;
+	mso-para-margin-bottom:8.0pt;
+	mso-para-margin-left:0in;
+	line-height:107%;
+	mso-pagination:widow-orphan;
+	font-size:11.0pt;
+	font-family:"Calibri",sans-serif;
+	mso-ascii-font-family:Calibri;
+	mso-ascii-theme-font:minor-latin;
+	mso-hansi-font-family:Calibri;
+	mso-hansi-theme-font:minor-latin;
+	mso-bidi-font-family:"Times New Roman";
+	mso-bidi-theme-font:minor-bidi;
+	mso-ansi-language:EN-US;
+	mso-fareast-language:EN-US;}
+</style>
+<!\\\[endif]-->
+
+<!--StartFragment-->
+
+### Introduction
+
+We know you love **HPE servers** for their **security** and **ease of management**. With the release of **iLO 7** on **Gen12 servers**, HPE has taken security to the next level. Unlike previous generations where **"production" mode** was the default, iLO 7 introduces **"secure standard" mode** by default, enhancing system security.
+
+Another major shift is in the default login method - **Virtual NIC (VNIC)** replaces the **CHiF interface**, which was used in **iLO 6 and earlier versions**.
+
+In this article, I’ll walk you through **managing iLO 7 using the iLORest 6.0 tool via in-band access**. We’ll start with **installing iLORest** on different operating systems and then dive into **logging into iLO 7**. Since **iLO 7 introduces a new application account (App account) login method**, I’ll also cover how iLORest 6.0 fully supports this feature.
+
+Let’s get started! 🚀
+
+<!--EndFragment-->
+
+**Installing iLORest 6.0 on Linux**
+
+On Linux, iLORest can be installed as an RPM package. If you already have a previous version installed, you can upgrade it using the -Uvh option.
+For a fresh installation, use:
+
+```bash
+rpm -ivh ilorest-6.0.x86_64.rpm
+```
+
+Here is a screenshot of successful installation.
+
+![](/img/rpm_linux.jpg)
+
+**App account creation during iLORest installation**
+
+During the RPM installation of iLORest 6.0, you might notice that the installer prompts for iLO credentials to create an App account. While this step is optional, HPE strongly recommends creating the App account during installation itself.
+Why?
+
+•	The App account provides an additional method for in-band authentication with iLO 7, enhancing security and flexibility.
+
+•	Once created, the App account allows you to log in without needing traditional credentials every time.
+
+If you choose to skip this step, you can always create the App account later using the following command:
+
+```bash
+ilorest appaccount create -u Administrator -p password –self
+```
+
+By leveraging this new authentication method, managing iLO 7 becomes even more seamless. 🚀
+
+**Installing iLORest 6.0 on Windows**
+
+On Windows, iLORest is installed using an MSI package. During installation, a user interface will appear, prompting you to enter iLO credentials for App account creation.
```

Comment:
* iLOrest
* Provide external link to the latest iLOrest packages (GitHub releases) .
* application account